Makefile

```diff
@@ -251,6 +251,9 @@ docker_local:
 docker_mini:
 	${call build_docker_image,docker/mini/Dockerfile,vitess/mini}
 
+docker_vttestserver:
+	${call build_docker_image,docker/vttestserver/Dockerfile.mysql57,vitess/vttestserver}
+
```

Comment:
We should build both `mysql57` and `mysql80` images and tag them so. Those are the tags we currently publish on dockerhub too.

Reply:
Sorry if my feedback wasn't clear. As it stands today `make docker_vttestserver` will produce an image like `vitess/vttestserver:latest`. This is confusing because you don't know which mysql flavor you are getting if you just go by the tag. It is in fact the `mysql57` flavor and that fact is embedded in the `Makefile`. To avoid this, we should build `mysql57` and `mysql80` images, and _not_ build a `latest` image at all.
Any change that requires rebuilding one of the images probably requires rebuilding the other. A given user may be using only one of those and not both. If that is what we want to support we should make that explicit.
So we either have two targets `docker_vttestserver_mysql57` and `docker_vttestserver_mysql80`, or just one target `docker_vttestserver` (which builds both images, similar to what you have written as the `docker_vttestserver_all` target).
Another option is to have **three** targets:
```
docker_vttestserver: builds 2 images
docker_vttestserver_mysql57
docker_vttestserver_mysql80
```
That is also acceptable.

Reply:
oh, I see what you mean 🤦  sorry about that Deepthi. 

I'll address this and attempt to add a test later today.